Automattic/jetpack
# Forms: Add Tabs to Dashboard inbox view

## Proposed changes:

* Add Tabs (Inbox|Spam|Trash) to Dashboard inbox view
* Also includes necessary changes from https://github.com/Automattic/jetpack/pull/29555

### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

Enable the jetpack forms package and the new dashboard before testing with:

```
add_filter( 'jetpack_contact_form_use_package', '__return_true' );
add_filter( 'jetpack_forms_dashboard_enable', '__return_true' );
```

- Go to `/wp-admin/admin.php?page=jetpack-forms`
- Check if the Inbox tabs are working properly
- Check if the Tabs and Actions menus are sticky to the top when scrolling
- Check if the Response details panel also sticks to the top when scrolling
